docs/proposals/user-cluster-mla.md

```diff
@@ -0,0 +1,335 @@
+# User-Cluster MLA (Monitoring, Logging & Alerting) & MLA Controller for KKP
+
+**Authors**: Rastislav Szabo (@rastislavs), Jiacheng Xu (@jiachengxu), Pavel Aborilov (@aborilov)
+
+**Status**: Draft proposal, [PoC](https://github.com/kubermatic/mla) in progress.
+
+**Issues**: https://github.com/kubermatic/kubermatic/issues/6321, https://github.com/kubermatic/kubermatic/issues/6397
+
+## Table of Contents
+
+- [Goals](#goals)
+- [Motivation and Background](#motivation-and-background)
+- [Implementation](#implementation)
+  - [User Cluster MLA Components](#user-cluster-mla-components)
+    - [Loki Promtail](#loki-promtail)
+    - [Prometheus](#prometheus)
+  - [Seed MLA Components](#seed-mla-components)
+    - [MLA Gateway](#mla-gateway)
+    - [Loki\-Distributed](#loki-distributed)
+    - [Cortex](#cortex)
```

Comment:
Why? What you choose Cortex over Thanos? What about the criteria back from a year ago in https://github.com/kubermatic/kubermatic/blob/master/docs/proposals/global-monitoring.md#options ? Do we really, really, REALLY want the gigantic complexity that is Cortex: https://github.com/cortexproject/cortex/blob/master/docs/architecture.md#chunk-store ?

Reply:
The most important reason that we choose Cortex over Thanos is that **Cortex provides a multi-tenancy alertmanager support but Thanos doesn't** :) This (Multi-tenancy) is the acceptance criteria for the User-Cluster MLA support
Also as you mentioned in `cons` in the global-monitoring proposal:
`Requires Cassandra/DynamoDB/BigTable for index storage`
This is already needed for Loki, so we will have this index storage for logs anyway, so we don't really see this as a big disadvantage for using Cortex.